src/npc.cpp

```diff
@@ -1064,6 +1067,28 @@ void npc::form_opinion( const player &u )
     add_msg( m_debug, "%s formed an opinion of u: %s", name, npc_attitude_id( attitude ) );
 }
 
+void npc::mutiny()
+{
+    bool seen = g->u.sees( pos() );
+    if( seen ) {
+        add_msg( m_bad, "%s is tired of your incompetent leadership and abuse!", disp_name() );
```

Comment:
Needs to be translatable - see "doc/TRANSLATING.md".

Reply:
Yes thanks!